### PR TITLE
Created controller, service and model for basic entries

### PR DESCRIPTION
**Pull request by j2law**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (j2law) [inline pom.xml:31]: This is needed for certain annotations such as Getters and Setters.

File: src/main/java/com/budgeteers/financetracker/api/controllers/entry/EntryController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package com.budgeteers.financetracker.api.controllers.entry;
```
Comment (j2law) [inline src/main/java/com/budgeteers/financetracker/api/controllers/entry/EntryController.java:1]: This EntryController class is the API for entry related actions/logic

File: pom.xml (hunk 1 above)
Comment (snowtheghost) [reply pom.xml:31]: Great job.

File: src/main/java/com/budgeteers/financetracker/api/controllers/health/HealthController.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/health")
+public class HealthController {
+
+    private static int counter = 0;
```
Comment (snowtheghost) [inline src/main/java/com/budgeteers/financetracker/api/controllers/health/HealthController.java:11]: This was for demonstration purposes. We can get rid of this counter logic in the next PR.

**Review by snowtheghost — APPROVED** [review]

**Pull request merged**